### PR TITLE
[BEAM-1487] Suppress flaky harness tests

### PR DESCRIPTION
**Pull request by tgroh**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by tgroh** [comment]
R: @dhalperi

File: sdks/java/harness/pom.xml [hunk 1]
```diff
@@ -44,6 +44,17 @@
           <target>1.8</target>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>org.apache.beam.fn.harness.logging.BeamFnLoggingClientTest</exclude>
```
Comment (dhalperi) [inline sdks/java/harness/pom.xml]: please add a link to the JIRA here.

**Comment by dhalperi** [comment]
LGTM, modulo one comment.

File: sdks/java/harness/pom.xml (hunk 1 above)
Comment (tgroh) [inline sdks/java/harness/pom.xml]: Done.

**Pull request closed without merging**